RELEASING.md

```diff
@@ -0,0 +1,29 @@
+# Instruction For Creating New Starknet Forge Releases
+
+1. Bump Starknet Forge versions in
+    1. Top-level `Cargo.toml`
+    2. `crates/forge/Cargo.toml`
+    3. `crates/cast/Cargo.toml`
+    4. Any other applicable user-facing tools
+2. Regenerate locks using `cargo generatae-lockfile`
+3. Merge introduced changes
+4. Create a new tag in repository with format `vMAJOR.MINOR.PATCH`
+5. Wait for release workflows to pass. A new draft release will be created on GitHub.
```

Comment:
Will release workflows start automatically? And when?

Reply:
It triggers on new tag creation